modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -409,6 +417,15 @@ export default class NestedTabs extends Base {
 			this.updateListeners( view );
 			elementor.$preview[ 0 ].contentWindow.dispatchEvent( new CustomEvent( 'elementor/elements/link-data-bindings' ) );
 		}
+
+		this.resetActiveTab();
+	}
+
+	resetActiveTab() {
+		const activeTabIndex = this.getActiveTabIndex() || 0;
```

Comment:
In some cases, the activate tab wasn't working correctly after using a repeater function.